librz/include/rz_util/rz_buf.h

```diff
@@ -235,7 +235,11 @@ DEFINE_RZ_BUF_WRITE_BLE(64)
 
 static inline bool rz_buf_read_offset(RZ_NONNULL RzBuffer *b, RZ_NONNULL RZ_INOUT ut64 *offset, RZ_NONNULL RZ_OUT ut8 *result, size_t size) {
 	rz_return_val_if_fail(b && offset && result, false);
-	if (rz_buf_read_at(b, *offset, result, size) != size) {
+	long long temp=rz_buf_read_at(b, *offset, result, size);
+	if(temp<0)
+		return false;
+	size_t temp_t=(size_t) temp;
+	if (temp != size) {
```

Comment:
Thanks for the contrib, in our project we always use brackets to define scopes (if/else/while/for/etc..)
You don't really need to make this changes, just cast inline.
Also on rizin we have declarations for `long long`, etc.. just use `st64` for signed.
I suggest to just change the line from
`if (rz_buf_read_at(b, *offset, result, size) != size) {`
to
`if (rz_buf_read_at(b, *offset, result, size) != (st64)size) {`

Reply:
Done